src/test/groovy/edu/ie3/datamodel/io/connectors/CsvFileConnectorTest.groovy

```diff
@@ -172,7 +170,7 @@ class CsvFileConnectorTest extends Specification {
 
   def "The csv file connector throws ConnectorException if no csv file definition can be built from class information"() {
     given:
-    def baseDirectory = tmpDirectory.toString()
+    def baseDirectory = tmpDirectory
```

Comment:
Maybe rename the field `tmpDirectory` to `baseDirectory`, so that this variable can be removed (same below)?